src/examples/20-seamless.html

```diff
@@ -18,10 +18,10 @@
             The seamless skin uses the height of your textarea as the minimum
             editor height.
         -->
-        <textarea class="wymeditor" style="min-height: 600px"></textarea>
+        <textarea class="wymeditor" style="min-height: 300px"></textarea>
```

Comment:
600 was too high. It required too many new paragraphs until it starts increasing height.